client/src/redux/constants/reportsConstants.js

```diff
@@ -1,6 +1,3 @@
-const GET_REPORT_DEPLOYMENTS = 'GET_REPORT_DEPLOYMENTS';
-const GET_REPORT_DEPLOYMENTS_CSV = 'GET_REPORT_DEPLOYMENTS_CSV';
-const GET_REPORT_DETAILS = 'GET_REPORT_DETAILS';
-const GET_REPORT_DETAILS_CSV = 'GET_REPORT_DETAILS_CSV';
+const GET_REPORT = 'GET_REPORT';
 
-export { GET_REPORT_DEPLOYMENTS, GET_REPORT_DEPLOYMENTS_CSV, GET_REPORT_DETAILS, GET_REPORT_DETAILS_CSV };
+export { GET_REPORT };
```

Comment:
Nice! 

Only thing that might come back around, a second constant with the `_CSV` suffix, but definitely leave this as is  for now, since as you pointed out we don't really use a bunch of distinction with state for reports